app/helpers/application_helper.rb

```diff
@@ -1363,11 +1363,13 @@ def user_role_name
     User.current_user.miq_user_role_name
   end
 
-  def rbac_common_feature_for_buttons(pressed)
+  def rbac_common_feature_for_buttons(pressed, resource_id = nil)
     # return feature that should be checked for the button that came in
     case pressed
     when "rbac_project_add", "rbac_tenant_add"
       "rbac_tenant_add"
+    when "rbac_tenant_manage_quotas"
+      MiqProductFeature.tenant_identifier('rbac_tenant_manage_quotas', resource_id)
```

Comment:
Would it make sense to use the same logic as in `assert_privileges` here?

(As in, don't hardcode `rbac_tenant_manage_quotas` if this should be happening for every `MiqProductFeature.my_root_tenant_identifier?(feature)` when `resource_id` is not nil?)

Reply:
all of this method is a hack

there was some generic mechanism and then someone added an "if" there for a particular button

now there's one more of such "if"s.

Then the UI team has to support this for ever.